crates/astria-conductor/src/celestia/mod.rs

```diff
@@ -353,37 +360,34 @@ impl RunningReader {
                 }
 
                 Some(res) = self.latest_heights.next() => {
-                    match res {
-                        Ok(height) => {
-                            info!(height, "observed latest height from Celestia");
-                            self.record_latest_celestia_height(height);
-                        }
-                        Err(error) => {
-                            warn!(
-                                %error,
-                                "failed fetching latest height from sequencer; waiting until next tick",
-                            );
-                        }
-                    }
+                    self.latest_height_handler(res);
                 }
 
             );
         };
 
         // XXX: explicitly setting the event message (usually implicitly set by tracing)
         let message = "shutting down";
-        match reason {
-            Ok(reason) => {
-                info!(reason, message);
-                Ok(())
+        report_exit(reason, message)
+    }
+
+    #[instrument(skip_all)]
+    fn latest_height_handler(&mut self, res: eyre::Result<u64>) {
```

Comment:
Maybe rename to `handle_latest_height` to be consistent with other methods?

Reply:
Renamed in 47b2a8d.